caluma/caluma_form/structure.py

```diff
@@ -38,6 +38,10 @@ def root(self):
         return self
 
     def get(self, name, default=None):
+        for original, aliases in self.aliases.items():
```

Comment:
`self.aliases` will only be defined for `Fieldset`. Not sure if the jexl will strictly only work on `FieldSet` but I would prefer either error handling here or having `aliases = {}` in `Element`.

Reply:
Good point, fixed